src/apps/chifra/pkg/file/output.go

```diff
@@ -0,0 +1,25 @@
+package file
+
+import (
+	"io"
+	"log"
+	"os"
+)
+
+// GetOutputFileWriter returns a writer which either the default one (same as passed
+// to this function) or, if filePath is non-empty, one that writes to that file.
+// If there is an error, log.Fatal is called, because there really is no good way
+// to recover. Plus, output file is disabled in server, so it is safe to exit.
+func GetOutputFileWriter(filePath string, defaultWriter io.Writer) io.Writer {
+	if filePath == "" {
+		// If there's no file path, we can just return the default writer
+		return defaultWriter
+	}
+
+	// Otherwise, let's try to create the file
+	file, err := os.Create(filePath)
```

Comment:
There's an Append option in the Globals data too. We use it in monitors. Should be a simple addition, just open the file with the right creation options.

Also, in general, I try to avoid using these 'convenience functions' from os package. If you look at this function (os.Create), it's a one-liner for ` OpenFile(name, O_RDWR|O_CREATE|O_TRUNC, 0666)`. In this way, I can search for O_RDWR and find all files opened for reading and writing, or O_CREATE vs O_APPEND to see which files we open for Append vs. destructive create.  I used this "ability to search" to make sure we are consistent in how we protect against destroying data (not finalized yet).

Upshot: use the expanded one-liner ` OpenFile(name, O_RDWR|O_CREATE|O_TRUNC, 0666)`

Reply:
I made this change.